adyenv6core/testsrc/com/adyen/v6/facade/AdyenCheckoutFacadeTest.java

```diff
@@ -376,4 +411,135 @@ private void setSuccessfulPaymentStubs() throws InvalidCartException {
         //updateOrder
         doNothing().when(adyenOrderService).updateOrderFromPaymentsResponse(any(), any());
     }
+
```

Comment:
can we add more test cases specifically for new order status PENDING_PAYMENT

Reply:
`testAuthorisePaymentRedirect` already goes through that flow. I added some more assertions to make it clear.